enterprise/api_client/discovery.py

```diff
@@ -72,6 +79,36 @@ def __init__(self, user):
         self.user = user
         self.client = course_discovery_api_client(user)
 
+    def get_search_results(self, querystring=None):
+        """
+        Return a search result from all data.
+
+        Returns:
+            dict: API response with search results
+
+        """
+        return self._load_data(
+            self.SEARCH_ALL_ENDPOINT,
+            default=[],
+            querystring=querystring,
+        )
+
+    def get_paginated_search_results(self, querystring=None):
+        """
+        Return paginated search result from all data.
+
+        Returns:
+            dict: API response with search results, as well as links to next and previous pages.
+
+        """
+        return self._load_data(
+            self.SEARCH_ALL_ENDPOINT,
+            default=[],
+            querystring=querystring,
+            traverse_pagination=False,
```

Comment:
Hehe sorry, I didn't forget, I just need to go figure it out myself :P I copied this from the other function that gets a paginated response. If I had to guess, it just means "instead of traversing all the pages and giving me everything, *don't* traverse them and thus give me a paginated response instead" -- but I'll tell you whether that's the case in my next comment.